src/common/actions/repositories.js

```diff
@@ -3,6 +3,7 @@ import 'isomorphic-fetch';
 import { conf } from '../helpers/config';
 
 const BASE_URL = conf.get('BASE_URL');
+const log = console;
```

Comment:
don't "get around" linting, just put in a comment to disable linting, either inline or doc wide, then it's clear. :)
```
// eslint-disable-line no-console
```